api/src/main/java/com/datastrato/gravitino/NameIdentifier.java

```diff
@@ -48,6 +50,18 @@ public static NameIdentifier of(Namespace namespace, String name) {
     return new NameIdentifier(namespace, name);
   }
 
+  /**
+   * Create the root {@link NameIdentifier}.
+   *
+   * <p>Note: this is a special {@link NameIdentifier} which has no namespace and name is "." and is
+   * only used for the tree lock intentionally.
+   *
+   * @return
+   */
+  public static NameIdentifier ofRoot() {
+    return ROOT;
+  }
```

Comment:
Don't add them into the API if it only used by you.

Reply:
Removed.